src/hermit.js

```diff
@@ -121,115 +53,41 @@ function commandMessage(data) {
  * @extends {module:interaction.IndirectKeystoreInteraction}
  */
 export class HermitInteraction extends IndirectKeystoreInteraction {
-}
-
-/**
- * Base class for interactions which read a QR code displayed by a
- * Hermit command.
- *
- * @extends {module:hermit.HermitInteraction}
- */
-export class HermitReader extends HermitInteraction {
-
-  constructor() {
-    super();
-    this.reader = true;
-  }
 
   messages() {
     const messages = super.messages();
     messages.push({
       state: ACTIVE,
       level: INFO,
       code: "hermit.scanning",
-      text: "Scan Hermit QR code now.",
+      text: "Scan Hermit QR code sequence now.",
     });
     return messages;
   }
 
 }
 
 /**
- * Base class for interactions which display data as a QR code for
- * Hermit to read and then read the QR code Hermit displays in
- * response.
+ * Reads a public key from data returned by Hermit's `display-xpub` command.
```

Comment:
think the comments got jacked up here -- it's a little hard to follow in the diff but -- it's not Reading a `public key` here